.github/workflows/production-suite.yml

```diff
@@ -90,7 +90,7 @@ jobs:
           sudo sed -i '/^bind/s/bind.*/bind 0.0.0.0/' /etc/redis/redis.conf
 
       - name: Build production tarball
-        run: mispipe "./tools/ci/production-build 2>&1" ts
+        run: ./tools/ci/production-build 2>&1
```

Comment:
This is reasonable but maybe we should add a quick mention in our `continuous-integration.md` doc that GitHub Actions supports this, so that folks know to look for it.

Reply:
Added more details on steps.